### ISSUE TITLE
Prep contribution limits snippet for 2019

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2535]
**What we're after**
Internal and external stakeholders will want access to the 2019-2020 contribution limits as soon as they're available. Right now we have a contribution limits snippet, but we're not using it. We need to make sure it's in great shape and inserted on the pages that use it.

**To do**
- [x]  ID the pages in H4CC that have it and make sure we've got a list of them stored in shared folder.
- [x] Review the snippet and make sure it's in great shape.
- [x] Insert the snippet where it goes in the H4CC pages identified above.

**Comment by dorothyyeager** [comment]
Pages are ID'ed in this document: https://docs.google.com/document/d/1A_qyq7sSYpafxaVKDzqYknBxqjE2CFSl7BBIX7qXaW0/edit?usp=sharing

For consistency, we should add the chart to a couple of pages that do not have it now (Contributions received by party; Contributions made by SSF and the Citizens info page), so I've included those.

**Comment by dorothyyeager** [comment]
I updated the snippet to also include the footnotes. The snippet has been added to all of the pages identified in the document. 

We will want to update the snippet for the 2019-2020 limits. At that time we'll want to make an archive of the previous cycle's table. I'll make a ticket for this work in early 2019.

For now, though, we're all prepared and updated, contribution limits table-snippet wise, and work is completed on this issue.
